File: pgm/ABC_Java/ABC_285_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final int a = Integer.parseInt (arr[0]);
        final int b = Integer.parseInt (arr[1]);
        final String Yes = "Yes";
        final String No = "No";
        if (a * 2 == b || a * 2 + 1 == b) {
            System.out.println (Yes);
        } else {
            System.out.println (No);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_285_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val one: Int = 1
const val two: Int = 2

fun main() {
    val (a: Int, b: Int) = readln().split(space).map { it.toInt() }
    if (a * two == b || a * two + one == b) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}